src/providers/export/tensorFlowPascalVOC.ts

```diff
@@ -155,16 +197,147 @@ item {
         const annotationsFolderName = `${exportFolderName}/Annotations`;
         await this.storageProvider.createContainer(annotationsFolderName);
 
+        const annotationTemplate = `
+<annotation verified="yes">
+    <folder>Annotation</folder>
+    <filename>%FILE_NAME%</filename>
+    <path>%FILE_PATH%</path>
+    <source>
+        <database>Unknown</database>
+    </source>
+    <size>
+        <width>%WIDTH%</width>
+        <height>%HEIGHT%</height>
+        <depth>3</depth>
+    </size\>
+    <segmented>0</segmented>
+    %OBJECTS%
+</annotation\>`;
+
+        const objectTemplate = `
+<object>
+    <name>%OBJECT_TAG_NAME%</name>
+    <pose>Unspecified</pose>
+    <truncated>0</truncated>
+    <difficult>0</difficult>
+    <bndbox>
+        <xmin>%OBJECT_TAG_xmin%</xmin>
+        <ymin>%OBJECT_TAG_ymin%</ymin>
+        <xmax>%OBJECT_TAG_xmax%</xmax>
+        <ymax>%OBJECT_TAG_ymax%</ymax>
+    </bndbox>
+</object>`;
+
+        const allAnnotationExports = [];  // Promise<void>[]
+
         // Save Annotations
-        // TODO
+        this.imagesInfo.forEach((imageInfo, imageName) => {
+            allAnnotationExports.push(
+                new Promise(async (resolve, reject) => {
+                    const imageFilePath = `${annotationsFolderName}/${imageName}`;
+                    const assetFilePath = `${imageFilePath.substr(0, imageFilePath.lastIndexOf("."))
+                        || imageFilePath}.xml`;
+
+                    const objectsXML = imageInfo.objects.map((o) => {
```

Comment:
If you convert the `%variable% notation to use javascript `${variable}` notation you can then use the same string interpolation function that we use in localization.